tests/model_monitoring/test_schemas.py

```diff
@@ -71,35 +71,27 @@ def test_fqn_parsing(
 @pytest.mark.parametrize(
     ("flat_mep", "expectation"),
     [
-        ({"project": "proj-1", "uid": "ok_30"}, does_not_raise()),
-        (
-            {},
-            pytest.raises(
-                pydantic.v1.ValidationError,
-                match=re.escape(
-                    "2 validation errors for ModelEndpointMetadata\nproject\n  field required "
-                    "(type=value_error.missing)\nuid\n  field required (type=value_error.missing)"
-                ),
-            ),
-        ),
```

Comment:
This test param (case) can be kept, just need to change the message or simple remove the `match=`